### PR TITLE
Issue29

### PR DESCRIPTION
**Pull request by Cruisererick**

### CODE REVIEW

File: junit/ScratchLoaderTest.java [hunk 1]
```diff
@@ -168,4 +173,144 @@ public void testUnzip()
...
+        Sprite sprite = new Sprite(name, spriteiScripts);
+        assertEquals(2, sprite.countScripts());
+    }
+
+     /**
+    * Test to get the length of the scripts inside the sprite.
+    *
+    */
+    @Test
+    public void make()
```
Comment (campellcl) [inline junit/ScratchLoaderTest.java]: We might want to re-name this method it's name is a little ambiguous.

File: junit/ScratchLoaderTest.java [hunk 2]
```diff
@@ -168,4 +173,144 @@ public void testUnzip()
...
+	project = null;
+        assertEquals(4 , count);
+    }
+
+     /**
+    * Test to get the length of the scripts inside the scratch project.
+    *
+    */
+    @Test
+    public void testgetTotalScriptLenght()
```
Comment (campellcl) [inline junit/ScratchLoaderTest.java]: Type-o in method name.

File: scratchgrader/ScratchGrader.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        }
+        return count;
+    }
+
+    /**
+     * getTotalScriptLenght - Get the total lenght between \
+     * all the scripts the of the project.
+     * @return int - total lenght of all scripts
+     */
+    public int getTotalScriptLenght()
```
Comment (campellcl) [inline scratchgrader/ScratchGrader.java:91]: Type-o in method name.

**Review by campellcl — CHANGES_REQUESTED** [review]
Looks good logic wise; just a couple of spelling type-o's and one vague method name

**Comment by cambelem** [comment]
Looks good with the new commits. Go ahead and merge.

**Pull request merged**